func.rb

```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class Func < Formula
+  v = "v0.17.1"
+  plugin_name = "func"
+  path_name = "kn-plugin-#{plugin_name}"
+  file_name = "#{plugin_name}"
+  base_url = "https://github.com/knative-sandbox/#{path_name}/releases/download/#{v}"
+
+  homepage "https://github.com/knative-sandbox/#{path_name}"
+
+  version v
+
+  if OS.mac?
+    url "#{base_url}/#{file_name}_darwin_amd64.gz"
+    sha256 "f83d1ff15045cd882fedda4944581d5e3084a82d68a309b4c5c900af91968e11"
+  else
+    url "#{base_url}/#{file_name}_linux_amd64.gz"
```

Comment:
Just curious - does this also work for Windows Subsystem for Linux?

Reply:
tbh, no idea whether brew is available for Windows Subsystem. Afaik Windows leverages a different package manager, https://chocolatey.org/. We might consider supporting this, too (but I never worked with it)

Reply:
IMO it should work with Windows Subsystem for Linux, once you'd have Homebrew installed. A while ago I've used native Windows manages that was in beta back then, so.... We might want to open that Pandora box. :)

Reply:
Seems like it should just work? https://medium.com/@edwardbaeg9/using-homebrew-on-windows-10-with-windows-subsystem-for-linux-wsl-c7f1792f88b3